src/content/docs/alerts/create-alert/create-alert-condition/create-nrql-alert-conditions.mdx

```diff
@@ -716,19 +717,29 @@ Loss of signal settings include a time duration and two possible actions.
+
+<Callout variant="important">
+  In order to prevent a loss of signal incident from opening when "Do not open "lost signal" incident on expected termination", the tag `termination: expected` must be added to the entity. This tag tells us the signal was expected to terminate. See how to add the tag [directly to the entity](/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data/#add-tags).
+</Callout>
 
 To create a NRQL alert configured with loss of signal detection in the UI:
 
-1. For a policy, when you create a condition, under <DNT>**Select a product**</DNT>, click <DNT>**NRQL**</DNT>, then click <DNT>**Next, define thresholds**</DNT>.
-2. Write a [NRQL query](/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries#syntax) that returns the values you want to alert on.
-3. For <DNT>**Threshold type**</DNT>, select <DNT>**Static**</DNT> or <DNT>**Anomaly**</DNT>.
-4. Click <DNT>**+ Add lost signal threshold**</DNT>, then set the signal expiration duration time in minutes or seconds in the <DNT>**Consider the signal lost after**</DNT> field.
-5. Choose what you want to happen when the signal is lost. You can check one or both of <DNT>**Close all current open incidents**</DNT> and <DNT>**Open new "lost signal" incident**</DNT>. These control how loss of signal incidents will be handled for the condition.
-6. Make sure you name your condition before you save it.
+1. Follow the instructions to create a [NRQL alert condition](/docs/alerts/create-alert/create-alert-condition/alert-conditions/#set-your-signal-behavior).
+2. On the <DNT>Set thresholds</DNT> [step](/docs/alerts/create-alert/create-alert-condition/alert-conditions/#thresholds) you'll find the option to <DNT>Add lost signal threshold</DNT>. Click this button.
+3. Set the signal expiration duration time in minutes or seconds in the <DNT>**Consider the signal lost after**</DNT> field.
+5. Choose what you want to happen when the signal is lost. You can check any or all of the following options: <DNT>**Close all current open incidents**</DNT>,  <DNT>**Open new "lost signal" incident**</DNT>, <DNT>**Do not open "lost signal" incident on expected termination**</DNT>. These control how loss of signal incidents will be handled for the condition.
+6. You can optionally add or remove static/anomaly numeric thresholds. A condition that has only a loss of signal threshold and no static/anomaly numeric thresholds is valid and is considered a "stand alone" loss of signal condition.
+6. Continue through the steps to save your condition.
+7. If you selected "<DNT>**Do not open "lost signal" incident on expected termination**</DNT>", you must add the `termination: expected` tag to the entity to prevent a loss of signal incident from opening. See how to add the tag [directly to the entity](/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data/#add-tags).
```

Comment:
```suggestion
7. If you selected "<DNT>**Do not open "lost signal" incident on expected termination**</DNT>", you must add the `termination: expected` tag to the entity to prevent a loss of signal incident from opening. See [how to add the tag directly to the entity](/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data/#add-tags).
```